src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -7067,5 +7067,125 @@ public void LinqQueryInConditionalExpression_Incomplete()
             }
             EOF();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/80537")]
+        public void ObjectInitializerWithColonInsteadOfEqualsSign()
+        {
+            var source = """
+class Class1
+{
+    public int X { get; set; }
+}
+
+class Program
+{
+    static void Main()
+    {
+        new Class1 { X: 0 };
+    }
+}
+""";
```

Comment:
please indent these one level more than the 'var source' line.

Reply:
please follow the pattern of the tests in this file.  In your case, you can just do `UsingExpression("new Class1 { X: 0 }")` and that will suffice.